com.chartboost.helium/Documentation/integration/integration-checker.md

```diff
@@ -0,0 +1,36 @@
+# Helium Integration Checker
+
+`HeliumIntegrationChecker.cs` is a utility class introduced in Helium Unity 3.3.0. The `HeliumIntegrationChecker` provides common utility methods to make developers life easier when integrating, updating, and testing the Helium Unity SDK. The following functionality can be found in it, and it is accessible through Unity Context Menus:
+
+* *`Helium/Integration/Status Check`*
+
+This will run a series of checks and test inside the Helium Unity SDK. It attempts to corroborate that any common integration issues are surfaced to developers with a popup window. It is recommended to developers to run this method once after every Helium Unity SDK update, as it will raise any issues that might need correction after package update is completed.
```

Comment:
```suggestion
This will run a series of checks and tests inside the Helium Unity SDK. It attempts to corroborate that any common integration issues are surfaced to developers with a popup window. It is recommended for developers to run this method once after every Helium Unity SDK update, as it will arise any issues that might need correction after the package update has completed.
```